dali_mqtt_daemon.py

```diff
@@ -223,31 +196,46 @@ def on_message_brightness_cmd(mqtt_client, data_object, msg):
         ).group(1)
     )
     try:
-        level = int(msg.payload.decode("utf-8"))
-        if not 0 <= level <= 255:
-            raise ValueError
-        logger.debug("Set light <%s> brightness to %s", light, level)
-        data_object["driver"].send(gear.DAPC(address.Short(light), level))
-        if level == 0:
-            # 0 in DALI is turn off with fade out
+        if light not in data_object["all_lamps"]:
+            raise KeyError
+        lamp_object = data_object["all_lamps"][light]
+        try:
+            level = int(msg.payload.decode("utf-8"))
```

Comment:
lamp_object should be updated with new level (@level.setter)

Reply:
Done

Reply:
from this point on, we should probably use lamp_object.level (which has been validated) and not level

Reply:
Done